pydis_site/apps/content/resources/guides/pydis-guides/contributing/creating-bot-account.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Setting up a Bot Account
+description: How to set up a bot account.
+icon: fab fa-discord
+---
+1. Go to the [Discord Developers Portal](https://discordapp.com/developers/applications/).
+2. Click on the `New Application` button, enter your desired bot name, and click `Create`.
+3. In your new application, go to the `Bot` tab, click `Add Bot`, and confirm `Yes, do it!`
+4. Change your bot's `Public Bot` setting off so only you can invite it, save, and then get your **Bot Token** with the `Copy` button.
+> **Note:** **DO NOT** post your bot token anywhere public, or it can and will be compromised.
```

Comment:
Took me a second to parse it with the way it's currently worded.
```suggestion
> **Note:** **DO NOT** post your bot token anywhere public. If you do it can and will be compromised.
```